### ISSUE TITLE
384.15 - Apostrophe is not escaped in USB Device name resulting in Javascript error 

### ISSUE THREAD

**Issue by cowmix88** [RMerl/asuswrt-merlin.ng#460]
![image](https://user-images.githubusercontent.com/4701123/77401291-e2a9e480-6d69-11ea-9665-4460aa8a5dcf.png)

My USB device has an apostrophe in the device name for some odd reason which causing a JS error and prevents loading the usb device in the web ui folder list for usb applications. 

/aidisk/getfolderarray.asp

```html
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta HTTP-EQUIV="Pragma" CONTENT="no-cache">
<meta HTTP-EQUIV="Expires" CONTENT="-1">
<link rel="shortcut icon" href="images/favicon.png">
<link rel="icon" href="images/favicon.png">
<script>
var result = ['SanDisk' Cruzer Fit#0#1'];
var layer_order = '0';
var motion = 'gettree';
function get_folder_tree_success(){
parent.get_tree_items(result, motion);
}
</script>
</head>
<body onload="get_folder_tree_success();">
</body>
</html>
```

> **getfolderarray.asp:10 Uncaught SyntaxError: Unexpected identifier
getfolderarray.asp:18 Uncaught ReferenceError: get_folder_tree_success is not defined
    at onload (getfolderarray.asp:18)
onload @ getfolderarray.asp:18**

**Comment by cowmix88** [comment]
Is there a way to possible force the name for the USB device and not use the one provided by the chip on the device?

**Comment by Syst3mSh0ck** [comment]
This is a bug, but I would just remove the character in the Volume Label that's causing the problem for now.

**Comment by RMerl** [comment]
Should be fixed with af5c1e32c1ee25eb346d38444555706b86b962d4 .
